.gitignore

```diff
@@ -77,6 +77,7 @@ common/temp/
 common/autoinstallers/*/.npmrc
 **/.rush/temp/
 *.lock
+rush-logs/
```

Comment:
Is this because we're now dropping JSON files in here? Can these files instead go inside the `.rush` folder if they're never expected to be human-readable?

Reply:
Yup, log files get ignored with a separate ignore rule. I'll update